yt_dlp/extractor/tiktok.py

```diff
@@ -556,7 +590,7 @@ class TikTokUserIE(TikTokBaseIE):
             'title': '6820838815978423302',
             'thumbnail': r're:https://.+_1080x1080\.webp'
         },
-        'expected_warnings': ['Retrying']
+        'expected_warnings': ['Retrying', 'Creating a generic title instead']
```

Comment:
Is https://github.com/yt-dlp/yt-dlp/pull/3549#issuecomment-1108589347 applicable here?